server/tests/lib/nl/detection/llm_api_test.py

```diff
@@ -106,16 +106,19 @@ class TestParseResponse(unittest.TestCase):
   def test_chat(self, input, want):
```

Comment:
should test_chat and test_text_block be updated to test_gemini and test_palm? and then use the respective extraction functions in each function?

Reply:
test_chat has both palm and gemini, but renamed them slightly.